python_codes/contour_plots.py

```diff
@@ -59,37 +64,36 @@
 
 fig,ax=subplots(1,1)
 
-contour_plot_temp_var = ax.contourf(Time, Pressure, sum(temp_variations, axis =2))
+contour_plot_temp_var = ax.contourf(Time, Pressure, mean(temp_variations, axis =2))
 title(r'Temperature Variations')
 ax.set_xlabel(r'Time(s)',fontsize = 12)
 ax.set_ylabel(r'Pressure',fontsize = 12)
-ax.set_ylim(max(pavg), min(pavg))
+ax.set_ylim(max(pavg), 10**6)
 ax.set_yscale('log')
 fig.colorbar(contour_plot_temp_var)
-savefig('contour_plot_temp.png')
+savefig('contour_plot_temp_deep.png')
 
 fig,ax=subplots(1,1)
 
-contour_plot_H2O = ax.contourf(Time, Pressure, sum(H2O_variations, axis = 2))
+contour_plot_H2O = ax.contourf(Time, Pressure, sum(H2O_variations_from_initial, axis = 2))
```

Comment:
Normally, we use lower case words for a variable. so change Time to **time**, Pressure to **pres**